src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphDAO.scala

```diff
@@ -221,20 +238,19 @@ trait GraphDAO {
         vertex.removeProperty(key)
         vertex.getVertices(Direction.OUT, key).foreach(removeMapVertex)
       case (key, Some(value: DateTime)) => vertex.setProperty(key, value.toDate)
-      case (key, Some(value: Map[_,_])) => serializeAttributeMap(vertex, key, value.asInstanceOf[Map[String, Attribute]], graph, workspaceName)
-      case (key, Some(value: AttributeEntityReference)) => serializeReference(vertex, key, value, graph, workspaceName)
+      case (key, Some(value: Map[_,_])) => serializeAttributeMap(vertex, key, value.asInstanceOf[Map[String, Attribute]], graph, workspaceContext)
```

Comment:
or ooo lets make it an implicit argument to 
![image](https://cloud.githubusercontent.com/assets/755311/9256763/6b49f180-41bf-11e5-88a2-a5d48e02a285.png)